src/lib.rs

```diff
@@ -134,7 +134,7 @@ impl std::default::Default for Assert {
         Assert {
             cmd: vec!["cargo", "run", "--"]
                 .into_iter().map(String::from).collect(),
-            expect_success: true,
+            expect_success: None,
```

Comment:
If we want to keep the "expect success", this is the only line that needs to change. (To `Some(true)`)